internal/kokoro/continuous.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+##
+# continuous.sh
+# Runs CI checks for entire repository.
+#
+# Jobs types
+#
+# Continuous: Runs root tests & tests in submodules changed by a PR. Triggered by PR merges.
+# Nightly: Runs root tests & tests in all modules. Triggered nightly.
+# Nightly/$MODULE: Runs tests in a specified module. Triggered nightly.
+##
+
+export GOOGLE_APPLICATION_CREDENTIALS=$KOKORO_KEYSTORE_DIR/72523_go_integration_service_account
+# Removing the GCLOUD_TESTS_GOLANG_PROJECT_ID setting may make some integration
+# tests (like profiler's) silently skipped, so make sure you know what you are
+# doing when changing / removing the next line.
+
+export GCLOUD_TESTS_GOLANG_PROJECT_ID=dulcet-port-762
+export GCLOUD_TESTS_GOLANG_KEY=$GOOGLE_APPLICATION_CREDENTIALS
+export GCLOUD_TESTS_API_KEY=`cat $KOKORO_KEYSTORE_DIR/72523_go_gcloud_tests_api_key`
+export GCLOUD_TESTS_GOLANG_KEYRING=projects/dulcet-port-762/locations/us/keyRings/go-integration-test
+export GCLOUD_TESTS_GOLANG_PROFILER_ZONE="us-west1-b"
+
+# Bigtable integration tests expect an existing instance and cluster
+#  ❯ cbt createinstance gc-bt-it-instance gc-bt-it-instance \
+#    gc-bt-it-cluster us-west1-b 1 SSD
+export GCLOUD_TESTS_BIGTABLE_KEYRING=projects/dulcet-port-762/locations/us-central1/keyRings/go-integration-test-regional
```

Comment:
Do the test use this?

Reply:
If you grep the source code we should be able to easily see if this value is sourced. If not it can be removed.

Reply:
_facepalm_ yep, good call.